spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::PersonalDetailsForm, type: :model do
+  describe '#name' do
+    it 'concatenates the first name and last name ' do
+      personal_details = FactoryBot.build(:personal_details_form, first_name: 'Bruce', last_name: 'Wayne')
+
+      expect(personal_details.name).to eq('Bruce Wayne')
+    end
+  end
+
+  describe '#main_language_english_options' do
```

Comment:
Non-blocking: perhaps not necessary to test this, as there's no logic inside the method. The integration test should cover the functionality I think?

Reply:
True, but now that it's there I guess there's not much harm in leaving it in...